code/game/objects/structures/vehicles/janicart.dm

```diff
@@ -111,7 +111,7 @@
 			tile.clean_blood()
 			for(var/A in tile)
 				if(istype(A, /obj/effect))
-					if(istype(A, /obj/effect/rune) || istype(A, /obj/effect/decal/cleanable) || istype(A, /obj/effect/overlay))
```

Comment:
You'll also have to exclude this from the janiborg, mop, and maybe a third thing

Reply:
https://github.com/vgstation-coders/vgstation13/blob/Bleeding-Edge/code/modules/detectivework/footprints_and_rag.dm#L56